### ISSUE TITLE
Steam Big Picture Segmentation fault 

### ISSUE THREAD

**Issue by BloodBlight** [LizardByte/Sunshine#604]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your issue described in the documentation?

- [X] I have read the documentation

### Is your issue present in the nightly release?

- [X] This issue is present in the nightly release

### Describe the Bug

Desktop apps seem to work fine (user input included), but attempting to launch the Steam Big Picture mode it crashes with a seg fault.

Tried both with steam already running, and it closed.

Thoughts?

### Expected Behavior

Steam should launch.

### Host Operating System

None

### Operating System Version

Linux Mint 20.04

### Architecture

64 bit

### Sunshine commit or version

v0.16.0

### Package

Linux - AppImage

### GPU Type

AMD

### GPU Model

AMD Radeon RX 580 2048SP

### GPU Driver/Mesa Version

Mesa 22.3.0~rc1+git2211041101.6d38a9768f8~f~mesarc2

### Capture Method (Linux Only)

X11

### Relevant log output

```Shell
[2022:12:21:23:42:31]: Info: Adding avahi service Sunshine
[2022:12:21:23:42:31]: Info: Configuration UI available at [https://localhost:47990]
[2022:12:21:23:42:32]: Info: Avahi service Sunshine successfully established.
[2022:12:21:23:42:34]: Info: /CN=NVIDIA GameStream Client -- verified
[2022:12:21:23:42:35]: Info: /CN=NVIDIA GameStream Client -- verified
[2022:12:21:23:42:36]: Info: /CN=NVIDIA GameStream Client -- verified
[2022:12:21:23:42:36]: Info: /CN=NVIDIA GameStream Client -- verified
/tmp/.mount_sunshiIYFfdo/AppRun: line 110: 893124 Segmentation fault      (core dumped) "$SUNSHINE_BIN_HERE" $@
```

**Comment by ReenigneArcher** [comment]
Share your `apps.json` file.

**Comment by BloodBlight** [comment]
Just the default I think:

<pre>
{
  "env": {
    "PATH": "$(PATH):$(HOME)/.local/bin"
  },
  "apps": [
    {
      "name": "Low Res Desktop",
      "prep-cmd": [
        {
          "do": "xrandr --output HDMI-1 --mode 1920x1080",
          "undo": "xrandr --output HDMI-1 --mode 1920x1200"
        }
      ]
    },
    {
      "name": "Steam BigPicture",
      "output": "steam.txt",
      "detached": [
        "setsid steam steam://open/bigpicture"
      ],
      "image-path": "steam.png"
    }
  ]
}
</pre>

**Comment by eldar** [comment]
I experience the same problem. I am on Ubuntu 22.10, NVIDIA RTX 2080Ti, Sunshine v.16.0. When attempting to launch Steam Big Picture, Sunshine host crashes with segmentation fault.

**Comment by SektorDV** [comment]
Happens to me too on AMD Radeon 5700XT, on Ubuntu 20.04, but it happens both in desktop and big picture. The stream usually runs for some 5 minutes and then terminates, with the log showing a segfault.

**Comment by BloodBlight** [comment]
This APPEARS to be resolved in v0.17, can anyone else confirm?

**Comment by ambientsound** [comment]
Steam Big Picture crashes for me with segfault 100% of the time, but Sunshine is pretty stable.
I'm on Radeon 5700 XT, Arch Linux, nightly build. Tried 0.17 but still crashing.
Launching apps directly with `setsid steam steam://rungameid/XXX` works the majority of the time.
